aqua gr causes panic

### aqua info

```console
$ aqua -v
aqua version 2.45.1 (d11aa12df4beb27c3370ce4ae999b1416aea312f)
```


### Overview

`aqua gr` command causes a panic.

### How to reproduce

aqua-generate-registry.yaml

```yaml
name: grpc/grpc-go/protoc-gen-go-grpc
version_prefix: cmd/protoc-gen-go-grpc/
```

Executed command and output

aqua-registry

```console
$ cmdx s -c aqua-generate-registry.yaml grpc/grpc-go/protoc-gen-go-grpc
+ set -eu

PACKAGE=${PACKAGE#https://github.com/}

bash scripts/check_commands.sh
bash scripts/check_gpgsign.sh
bash scripts/check_diff.sh
bash scripts/checkout.sh "$PACKAGE"

if [ "$IS_RECREATE" = true ]; then
  aqua exec -- cmdx rm
fi

bash scripts/start.sh
bash scripts/scaffold.sh "$PACKAGE" "$CMD" "$LIMIT" "$CONFIG"
aqua exec -- cmdx gr
bash scripts/commit.sh "$PACKAGE"
bash scripts/test.sh "$PACKAGE"
bash scripts/start.sh aqua-registry-windows
bash scripts/test-windows.sh "$PACKAGE"

From https://github.com/aquaproj/aqua-registry
 * branch                main       -> FETCH_HEAD
 * [new branch]          main       -> temp-remote-20250315175627/main
branch 'feat/grpc/grpc-go/protoc-gen-go-grpc' set up to track 'temp-remote-20250315175627/main'.
Switched to a new branch 'feat/grpc/grpc-go/protoc-gen-go-grpc'
[INFO] Checking if the container aqua-registry exists
[INFO] Checking if the container aqua-registry is running
[INFO] Dockerfile isn't updated
Successfully copied 2.05kB to aqua-registry:/workspace/scaffold.yaml
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.groupByExcludedAsset({0x40000aed00, 0x0, 0x8})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:222 +0x46c
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.sortAndMergeGroups({0x40000aecc0, 0x8, 0x9bb8e0?})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:199 +0x1d0
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).group(0x4000171400, 0x40004430a0, {0xffffd821adff, 0x1f}, {0x400041fb88?, 0x997520?, 0x1?})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:282 +0x4ec
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).generatePackage(0x40004430a0?, 0x4000039410?, 0x400037f188, {0xffffd821adff?, 0x4000322378?}, {0x400041fb88?, 0x400057e168?, 0x40005937b0?})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/group.go:286 +0x3c
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfoWithVersionOverrides(0x4000171400, {0xb99bf0, 0x40004363c0}, 0x40004430a0, {0xffffd821adff, 0x1f}, 0x400037f188, 0x857020?, 0x40003231e8)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/version_overrides.go:149 +0x514
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfoMain(0x4000171400, {0xb99bf0, 0x40004363c0}, 0x40004430a0, {0xffffd821adff?, 0x0?}, 0x0, 0x40003231e8)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:158 +0x50c
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).getPackageInfo(0x4000323148?, {0xb99bf0?, 0x40004363c0?}, 0xb9ec60?, {0xffffd821adff?, 0x0?}, 0x40003232b8, 0x4000436b40?)
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:112 +0x38
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).genRegistry(0x4000171400, {0xb99bf0, 0x40004363c0}, 0x40003232b8, 0x40003231e8?, 0x400012d108?, {0xffffd821adff, 0x1f})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:71 +0x54
github.com/aquaproj/aqua/v2/pkg/controller/generate-registry.(*Controller).GenerateRegistry(0x4000171400, {0xb99bf0, 0x40004363c0}, 0x40003232b8, 0x40004430a0, {0x40004666d0?, 0x1?, 0x1?})
        /home/runner/work/aqua/aqua/pkg/controller/generate-registry/generate.go:47 +0x16c
github.com/aquaproj/aqua/v2/pkg/cli/genr.(*command).action(0x400011a870, 0x4000436800)
        /home/runner/work/aqua/aqua/pkg/cli/genr/command.go:145 +0x210
github.com/urfave/cli/v2.(*Command).Run(0x4000458840, 0x4000436800, {0x4000100960, 0x6, 0x6})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/command.go:276 +0x600
github.com/urfave/cli/v2.(*Command).Run(0x4000458f20, 0x40004364c0, {0x4000128000, 0x7, 0x7})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/command.go:269 +0x83c
github.com/urfave/cli/v2.(*App).RunContext(0x40002f2800, {0xb99bf0, 0x40004363c0}, {0x4000128000, 0x7, 0x7})
        /home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.6/app.go:333 +0x51c
github.com/aquaproj/aqua/v2/pkg/cli.Run({0xb99bf0, 0x40004363c0}, 0x4000171310, {0x4000128000, 0x7, 0x7})
        /home/runner/work/aqua/aqua/pkg/cli/runner.go:115 +0x7b8
main.core(0x40004430a0, 0x4000444000)
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:45 +0x23c
main.main()
        /home/runner/work/aqua/aqua/cmd/aqua/main.go:31 +0x48
exit status 2
```


### Expected behaviour

`aqua gr` succeeds.

### Actual behaviour

`aqua gr` command causes a panic.

### Note

- https://github.com/aquaproj/aqua-registry/issues/33247

--- a/pkg/controller/generate-registry/group.go
+++ b/pkg/controller/generate-registry/group.go
@@ -218,6 +218,9 @@ func excludeGroupsAssets(groups []*Group, pkgName string) {
 }
 
 func groupByExcludedAsset(groups []*Group) []*Group {
+	if len(groups) == 0 {
+		return groups
+	}
 	newGroups := make([]*Group, 0, len(groups))
 	prevGroup := groups[0]
 	for _, group := range groups[1:] {

